CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -72,6 +72,13 @@ function Patient:onClick(ui, button)
         ui:addWindow(UIPatient(ui, self))
       end
     end
+  --[[ Support for knocking over patients, currently disabled
+  elseif button == "right" and not self.world:isPaused() then
```

Comment:
Possibly something better than `self.world:isPaused()`

Reply:
Or maybe this check is in the wrong place?

Reply:
I've decided to move this down into the next if statement, to make the parent if cleaner.